src/__stories__/stacking-group-story.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import {skinVars} from '..';
+import {ThemeVariant} from '../theme-variant-context';
+import StackingGroup from '../stacking-group';
+import Image from '../image';
+import Avatar from '../avatar';
+import img from './images/avatar.jpg';
+
+export default {
+    title: 'Components/StackingGroup',
+    parameters: {
+        fullScreen: true,
+    },
+};
+
+type Args = {
+    stacked?: boolean;
+    type: 'circle' | 'square';
+    size: number;
+    maxItems?: number;
+    children?: React.ReactNode;
+    itemsToInclude: number;
+    inverse: boolean;
+};
+
+export const Default: StoryComponent<Args> = ({
+    stacked,
+    size,
+    type,
+    maxItems,
+    itemsToInclude,
+    inverse = false,
+}) => {
+    return (
+        <ThemeVariant isInverse={inverse}>
+            <div
+                style={{
+                    padding: 16,
+                    width: 'fit-content',
+                    background: inverse ? skinVars.colors.backgroundBrand : skinVars.colors.background,
+                    lineHeight: 0,
+                }}
+                data-testid="stacking-group"
+            >
```

Comment:
instead of applying the background color and the padding manually, you could use a `<ResponsiveLayout isInverse>` + `<Box>` here:
```js
<ResponsiveLayout isInverse={inverse}>
    <Box paddingY={24}>
```